library/math/dual.lisp

```diff
@@ -0,0 +1,176 @@
+;;;; dual.lisp
+
+"
```

Comment:
Good doc. Add it as the `:documentation` part of the `defpackage` so that it shows up on the doc generator. The docs for `Hash`, `Eq`, and `Ord` should be moved to the docstrings for their respective `define-instance` forms.